### ISSUE TITLE
GLib-GIO:       Settings schema 'br.com.bb.pw3270' is not installed

### ISSUE THREAD

**Issue by marlonchalegre** [PerryWerneck/pw3270#24]
I built the version available in the [Release](https://github.com/PerryWerneck/pw3270/archive/refs/tags/5.3.zip) section of this project.

After `make install` when I try to open the app I receive the following error message:

GLib-GIO:       Settings schema 'br.com.bb.pw3270' is not installed
[1]    35433 trace trap (core dumped)  pw3270

I built the software in Fedora 34 Workstation.

Could you help me with this?

**Comment by PerryWerneck** [comment]
I've to check but looks like the 'make install' is missing a call to 'glib-compile-schemas'. Mostly users install using the .rpm ou .deb file who does this automatically.

**Comment by marlonchalegre** [comment]
Yeah, I know. I was using the rpm package.

I started to get an error trying to connect to my provider[server?]. A few weeks ago  the app started to give me an error message about the ssl connection  (was not possible to negotiate... , unsupported protocol). 

Because of that I was looking for an older version to install and check if the problem also occurs. So, I found this github and tried to build.
